samples/net/sockets/echo_client/src/ca_certificate.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __CA_CERTIFICATE_H__
+#define __CA_CERTIFICATE_H__
+
+#define CA_CERTIFICATE_TAG 1
+
+#define TLS_PEER_HOSTNAME "localhost"
+
+/* This is the same cert as what is found in net-tools/echo-apps-cert.pem file
+ */
+static const unsigned char ca_certificate[] = {
```

Comment:
Could we have the pem file as a file and just emded it here by including it. So similar way as what is done for example in rpl_border_router sample.

Reply:
Sounds reasonable, I can give a try. I guess you ment der file though? We've agreed in review of #7118 to use DER cert format to save some code size.

Reply:
> I guess you ment der file though? We've agreed in review of #7118 to use DER cert format to save some code size.

Yes, I meant the DER file.